examples/http-proxy/http_proxy.py

```diff
@@ -28,8 +28,14 @@
     print_env_info,
 )
 
+# the timeout after we commission our service instances
+# before we abort this script
 STARTING_TIMEOUT = timedelta(minutes=4)
 
+# additional expiration margin to allow providers to take our offer,
+# as providers typically won't take offers that expire sooner than 5 minutes in the future
+EXPIRATION_MARGIN = timedelta(minutes=5)
```

Comment:
OK, we had a talk and now I more-or-less understand.
Still, I think this is hard to understand & I this is a serious problem. Although maybe not for this PR.

Reply:
we don't _add_ expiration to the Cluster, just use an already existing parameter in an example so that we can easily verify whether that functionality works or not